Marlin/Marlin_main.cpp

```diff
@@ -7499,11 +7499,11 @@ inline void gcode_M115() {
       SERIAL_PROTOCOLLNPGM("Cap:SOFTWARE_POWER:0");
     #endif
 
-    // TOGGLE_LIGHTS (M355)
+    // CASE LIGHTS (M355)
     #if HAS_CASE_LIGHT
-      SERIAL_PROTOCOLLNPGM("Cap:TOGGLE_LIGHTS:1");
+      SERIAL_PROTOCOLLNPGM("Cap:CASE_LIGHT_BRIGHTNESS:1");
     #else
-      SERIAL_PROTOCOLLNPGM("Cap:TOGGLE_LIGHTS:0");
+      SERIAL_PROTOCOLLNPGM("Cap:CASE_LIGHT_BRIGHTNESS:0");
```

Comment:
No I was not following this. This change is not good as you say it is not possible to toggle light on/off. Server e.g. detects Cap:TOGGLE_LIGHTS and if not present you can not control lights in interface. Currently We use only S0/S1 to toggle light on/off which is what this capability defines. g as you can still toggle it - and I would like to see it continued to work, you should still send Cap:TOGGLE_LIGHTS:1
The fact that you can adjust brightness is an additional feature, so if that is the case add the capability. That way if server ever learns to adjust brightness it can detect it and until then it will just toggle and user has to manually adjust brightness.

Also PLEASE keep S0/1 for toggle and not switch to T0/1 - hosts use S and is just a pain if such things change knowing there are 100 forks having S still then.